packages/client/src/services/MetricsService.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export default class MetricsService {
+  async sendMetric(metricName: string, path: string): Promise<void> {
```

Comment:
probably make path `?:` and deal with it being null in this method instead of every time it is called.

Reply:
Creating optional parameters in TS: https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters